### ISSUE TITLE
Update loss terms in example notebooks to Tensorboard

### ISSUE THREAD

**Issue by alizma** [METHODS-Group/DRDMannTurb#42]
Tensorboard isn't too nice with rendering in Sphinx -- Tensorboard has no support for Sphinx and no viable way to ensure interactivity in rendered documentation. Part of this is because it is necessary for the documentation machine to have a port open for Tensorboard visualizations, which cannot reliably (or safely) be ensured. It also seems that both TensorFlow and Pytorch end up embedding screenshots of Tensorboard into their documentation, which is fine, but requires some pipelining in the background. An example of how to do this is provided in ``./examples/plot_image.py`` for a simple example of how to do this in Sphinx. 

The following examples require updates to the loss plots: 
- [ ] Custom data fit 
- [ ] Interpolation and fitting 
- [ ] Synthetic data fit with 3 loss terms 
- [ ] Synthetic data fit with a dnn 
- [ ] Training and wind generation from pre-trained model
